x-pack/plugins/ml/public/settings/settings.test.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
+jest.mock('ui/chrome', () => ({
+  getBasePath: jest.fn()
+}));
+
+import { shallow, mount } from 'enzyme';
+import React from 'react';
+
+import { Settings } from './settings';
+
+
+describe('Settings', () => {
+
+  test('Renders settings page', () => {
+    const wrapper = shallow(
+      <Settings canCreateFilter={true}/>
```

Comment:
This will need updating too to use `canCreateCalendars`

Reply:
Added! 👍